airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/Debezium.kt

```diff
@@ -25,7 +25,10 @@ value class DebeziumRecordKey(val wrapped: JsonNode) {
 @JvmInline
 value class DebeziumRecordValue(val wrapped: JsonNode) {
 
-    /** True if this is a Debezium heartbeat event. These aren't passed to [DebeziumConsumer]. */
```

Comment:
`DebeziumConsumer` is not a thing in this codebase. I forgot to update this comment when, a few months ago, I copy-pasted this code from one of my prototypes from early this year.